scripts/website.js

```diff
@@ -52,29 +52,53 @@ markdown.setOptions({
     return highlight.highlight(code, { language }).value;
   }
 });
-markdown.use({ renderer });
-
-const testPath = path.resolve(cwd, 'test')
-
-const tests = [
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'geojson.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/transactions.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'schema.alias.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'model.middleware.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/date.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/lean.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/cast.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/findoneandupdate.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/custom-casting.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/getters-setters.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/virtuals.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/defaults.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/discriminators.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/promises.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/schematypes.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/validation.test.js')).toString()),
-  ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/schemas.test.js')).toString())
+
+const testPath = path.resolve(cwd, 'test');
+
+/** additional test files to scan, relative to `test/` */
+const additionalTestFiles = [
+  'geojson.test.js',
+  'schema.alias.test.js'
 ];
+/** ignored files from `test/docs/` */
+const ignoredTestFiles = [
+  // ignored because acquit does not like "for await"
+  'asyncIterator.test.js'
+];
+
+/**
+ * Load all test file contents with acquit
+ * @returns {Object[]} acquit ast array
+ */
+async function getTests() {
+  const promiseArray = [];
+
+  for (const file of additionalTestFiles) {
```

Comment:
Can we please combine these two into one loop, `for (const file of [...fs.readdirSync(testDocs), ...additionalTestFiles]) {}` and use `readFileSync()`? With no `flatMap()` at the end. Parallelization is nice, but I would prefer to optimize for concise readability in this code rather than perf; shaving off a few ms on a docs build isn't worth it IMO.

Reply:
is c0c408c8ac011cd15f483337ae2ec0b3add477fa better? i couldnt avoid the last `.flat()` because otherwise it would constantly create a new array in each loop